plugins/BEdita/Core/src/Model/Action/SignupUserAction.php

```diff
@@ -29,31 +32,35 @@
  *
  * @since 4.0.0
  */
-class SignupUserAction extends BaseAction
+class SignupUserAction extends BaseAction implements EventListenerInterface
 {
+
+    use EventDispatcherTrait;
     use MailerAwareTrait;
 
     /**
      * The UsersTable table
      *
      * @var \BEdita\Core\Model\Table\UsersTable
      */
-    protected $Users = null;
+    protected $Users;
 
     /**
      * The AsyncJobs table
      *
      * @var \BEdita\Core\Model\Table\AsyncJobsTable
      */
-    protected $AsyncJobs = null;
+    protected $AsyncJobs;
 
     /**
-     * {@inheritDoc}
+     * {@inheritdoc}
```

Comment:
why {@inheritdoc} instead of {@inheritDoc} ?
https://www.phpdoc.org/docs/latest/guides/inheritance.html#the-inheritdoc-tag
both should work, but the camelcase version seems correct